src/events/index.ts

```diff
@@ -20,8 +62,12 @@ async function processEvents() {
   const events = await db.queryAsync('SELECT * FROM events WHERE expire <= ?', [
     ts
   ]);
+
   console.log('Process event start', ts, events.length);
+
   for (const event of events) {
+    if (event.event === 'proposal/start') sendPushNotification(event);
```

Comment:
There should be a way to check if "SERVICE_PUSH_NOTIFICATIONS"  is enabled like this:
```const servicePushNotifications = parseInt(process.env.SERVICE_PUSH_NOTIFICATIONS || '0');```
```suggestion
    if (servicePushNotifications && event.event === 'proposal/start') sendPushNotification(event);
```